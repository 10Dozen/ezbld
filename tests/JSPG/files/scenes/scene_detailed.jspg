# DetailedScene | dialog my_character.jpg

*pre_exec: { MyGame.ExecutePre() }
*post_exec: { MyGame.ExecutePost() }
*goto: Scene2

This is a scene with properties.

It should be parsed into fully configured scene entity.
