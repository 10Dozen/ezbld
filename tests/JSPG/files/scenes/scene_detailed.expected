[0]
cls = JSPGScene
name = DetailedScene | TEXT
type = dialog | TEXT
portrait = my_character.jpg | TEXT
pre_exec = ()=>{ MyGame.ExecutePre() } | FUNCTION
post_exec = ()=>{ MyGame.ExecutePost() } | FUNCTION
goto = Scene2 | TEXT
desc =
    "This is a scene with properties."

    "It should be parsed into fully configured scene entity."
